changelog/0.8.2/pull-1549

```diff
@@ -0,0 +1,9 @@
+Enchancement: Speed up querying across indices and speed up scanning existing files
```

Comment:
Also, the title is too long, max is 80 chars. I'd suggest removing the second "speed up":
```
Enhancement: Speed up querying across indices and scanning existing files
```